packages/react-server/src/ReactFizzServer.js

```diff
@@ -5888,11 +5930,32 @@ function flushCompletedQueues(
 
       flushedByteSize = request.byteSize; // Start counting bytes
       // TODO: Count the size of the preamble chunks too.
```

Comment:
This TODO should probably be fixed since this can have meaningful impact on this metric.